pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -300,21 +304,20 @@ def test_merge_single_cells():
     )
 
     # Confirm that the merge correctly reversed the object number (opposite from Parent)
-    assert (
-        sc_merged_df.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cells.tolist()
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
+    assert_cols = [
+        "Metadata_ObjectNumber",
+        "Metadata_ObjectNumber_cytoplasm",
+        "Metadata_ObjectNumber_cells",
+    ]
+    # check that we have the same data using same cols, sort and a reset index
+    pd.testing.assert_frame_equal(
```

Comment:
this doesn't looked blacked to me, please confirm

Reply:
I ran black on this, and `--check` doesn't seem to flag it (may be profile related). I'm seeing that there's a list end bumping up against the comment, I'll add in a newline there with new changes. Are you seeing anything else that looks stylistically out of place which I may change?